js/main.js

```diff
@@ -385,28 +385,77 @@ function editDelNote(event) {
         $form.elements.notes.value = fieldNotes.edit.notes;
       } else if (event.target.getAttribute('class') === 'fas fa-minus-square icon-white') {
         $modal.className = 'modal view';
-        event.target.className = 'fas fa-minus-square icon-red';
+        event.target.className = 'fas fa-minus-square icon-red-review';
       }
     }
   }
 }
 
 function closeModal(event) {
-  $modal.className = 'modal view hidden';
-  var $redIcon = document.querySelector('.icon-red');
-  $redIcon.className = 'fas fa-minus-square icon-white';
+  var $redIcon;
+  if ($pageReview.getAttribute('class') === 'container view') {
+    $modal.className = 'modal view hidden';
+    $redIcon = document.querySelector('.icon-red-review');
+    $redIcon.className = 'fas fa-minus-square icon-white';
+  } else {
+    $modal.className = 'modal view hidden';
+    $redIcon = document.querySelector('.icon-red-gear');
+    $redIcon.className = 'fas fa-minus-square icon-gear-del';
+  }
 }
 
-function deleteNote(event) {
-  for (var i = 0; i < fieldNotes.notes.length; i++) {
-    if (fieldNotes.notes[i].noteNum === noteIDNum) {
-      fieldNotes.notes.splice(i, 1);
+function deleteNoteGear(event) {
+  if ($pageRecord.getAttribute('class') === 'container view') {
+    for (var i = 0; i < fieldNotes.notes.length; i++) {
+      if (fieldNotes.notes[i].noteNum === noteIDNum) {
+        fieldNotes.notes.splice(i, 1);
+      }
     }
-  }
-  var $fieldNotesChildren = $fieldNotes.childNodes;
-  for (i = 0; i < $fieldNotesChildren.length; i++) {
-    if ($fieldNotesChildren[i] === closestElement) {
-      $fieldNotesChildren[i].remove();
+    var $fieldNotesChildren = $fieldNotes.childNodes;
+    for (i = 0; i < $fieldNotesChildren.length; i++) {
+      if ($fieldNotesChildren[i] === closestElement) {
+        $fieldNotesChildren[i].remove();
+      }
+    }
+  } else {
+    for (i = 0; i < gearData.cameras.length; i++) {
+      if (gearData.cameras[i] === closestElement.textContent) {
+        gearData.cameras.splice(i, 1);
+      }
+    }
+    for (i = 0; i < gearData.lenses.length; i++) {
+      if (gearData.lenses[i] === closestElement.textContent) {
+        gearData.lenses.splice(i, 1);
+      }
+    }
+    for (i = 0; i < gearData.filters.length; i++) {
+      if (gearData.filters[i] === closestElement.textContent) {
+        gearData.filters.splice(i, 1);
+      }
+    }
```

Comment:
Is the reason there's multiple for loops here because you don't know what type of gear this is? You can probably reduce this to one loop if you can save what type of gear it is onto the element itself with a `data-` attribute

https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes